project/src/main/java/com/google/sps/servlets/EventDetailServlet.java

```diff
@@ -0,0 +1,22 @@
+package com.google.sps.servlets;
+
+import com.google.sps.data.Event;
+import com.google.sps.utilities.CommonUtils;
+import com.google.sps.utilities.SpannerTasks;
+import java.io.IOException;
+import java.util.Set;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/event-details")
```

Comment:
can we rename servlet to like 'discovery-event-details' or something like 'all-event-details'? to prevent from potential confusion for event-details.html servlet

Reply:
Done